pyrh/models/option.py

```diff
@@ -9,6 +9,36 @@
 from .base import BaseModel, BaseSchema
 
 
+class MinTicks(BaseModel):
+    """Min ticks data class. Describes min increments the option can be traded at."""
+
+    pass
+
+
+class MinTicksSchema(BaseSchema):
+    """Min ticks schema data loader."""
+
+    __model__ = MinTicks
+
+    above_tick = fields.Float()
+    below_tick = fields.Float()
+    cutoff_price = fields.Float()
+
+    @post_load
+    def make_object(self, data: JSON, **kwargs: Any) -> MinTicks:
```

Comment:
Actually this isn't needed since the base class defines this method. We only need to override this method call if we need to do something special in the method (e.g. extract values from a "results" key as in portfolio)

Reply:
Got it